ledfx/virtuals.py

```diff
@@ -339,7 +341,37 @@ def set_preset(self, preset_info):
         )
         self.set_effect(effect)
 
-    def set_effect(self, effect):
+    def set_fallback(self):
+        """
+        Sets the active effect to the last effect that was active before the current effect.
+        """
+        if self.last_active_effect_type is not None:
+            effect_config = {}
```

Comment:
This is lifted code from how the api finds the effect config against the virtual. Might be able to improve this, but it works. Maybe make a util function